test/integration/component/test_host.py

```diff
@@ -197,8 +194,8 @@ def test_01_host_ha_with_nfs_storagepool_with_vm(self):
             
 
         hostToTest = listHost[0]
-        hostUpInCloudstack = wait_until(10, 10, self.checkHostUp, hostToTest.ipaddress, hostToTest.ipaddress)
-        #hostUpInCloudstack = wait_until(40, 10, self.checkHostStateInCloudstack, "Up", hostToTest.id)
+        #hostUpInCloudstack = wait_until(10, 10, self.checkHostUp, hostToTest.ipaddress, hostToTest.ipaddress)

```

Comment:
What's the reason to change the test here?

Reply:
checkHostUp method only ping the Host and verify that host is available. However we need to check the host state as 'UP' so updated to check the host state.

Reply:
Can you delete the commented line?